src/table/hooks/use-layout-service.ts

```diff
@@ -0,0 +1,7 @@
+import { useMemo } from '@nebula.js/stardust';
+import createLayoutService from '../../services/layout-service';
+import { TableLayout, LayoutService } from '../../types';
+
+const useLayoutService = (layout: TableLayout): LayoutService => useMemo(() => createLayoutService(layout), [layout]);
```

Comment:
custom hooks that we do using startust hooks, we normally store in src/nebula-hooks

Reply:
Removed it!